src/qml/LayerTreeItemProperties.qml

```diff
@@ -199,12 +199,20 @@ Popup {
             text: qsTr("Opacity")
             font: Theme.defaultFont
           }
-          Slider {
+
+          QfSlider {
             Layout.fillWidth: true
+
             id: slider
-            value: index !== undefined ? layerTree.data(index, FlatLayerTreeModel.Opacity) : 0
-            onMoved: {
-              layerTree.setData(index, value, FlatLayerTreeModel.Opacity)
+            value: index !== undefined ? layerTree.data(index, FlatLayerTreeModel.Opacity) * 100 : 0
+            from: 0
+            to: 100
+            stepSize: 1
+            suffixText: "%"
```

Comment:
```suggestion
            suffixText: " %"
```

Proposing a nonbreaking space

Reply:
> There are various spacing rules for the percentage sign depending on the language being used (according to the writers who do [research paper services for Ultius](https://www.ultius.com/writing-services/research-paper-writing-services.html)). In English the percent sign and the number is corresponds to are not separated by space, but the [International System of Units](https://www.nist.gov/programs-projects/international-system-units-si) and ISO 31-0 standard require a non-breaking space between them.

> The Chinese percent sign never uses space because Chinese characters and words are never separated. In Swedish, German, French, Finnish, and Czech there is a non-breaking space between the number and percent sign. In Italian, the percent sign is never spaced. In Persian and Turkish, the percent sign precedes the number instead of following it (for example %3). It is possible that the 3% we know today evolved from using the slash sign in Latin-based languages prior to the use of the decimal point (an example is 1/100th or 1% written as 0/01).  Perhaps 0/0 evolved into the percent sign we know today: %.

I like it more without the space, as in English. But let's conform the ISO...